HIP/hip-991.md

```diff
@@ -121,21 +123,25 @@ We propose adding a fixed fee mechanism to the Hedera Consensus Service (HCS) fo
 * FEKL has a maximum of `MAX_ENTRIES_FOR_FEE_EXEMPT_KEY_LIST` entries.
 * If a TopicMessageSubmitTransaction is submitted to the network and it contains a signature from a key included in the FEKL, no fees will be charged for that message.
   
-* **Handling Duplicates**:
-   - If the FEKL list contains duplicate keys, the transaction will fail with an `FEKL_CONTAINS_DUPLICATED_KEYS` error. This ensures that duplicate entries are not silently ignored, preventing potential bugs or issues in the calling code.
+##### Handling Duplicates
+
+If the FEKL list contains duplicate keys, the transaction will fail with an `FEKL_CONTAINS_DUPLICATED_KEYS` error. This ensures that duplicate entries are not silently ignored, preventing potential bugs or issues in the calling code.
+
+##### Signatures for Invalid/Inactive/Deleted Accounts
+
+The FEKL list will only require keys to be formally valid as per protobuf specifications, meaning they must be correctly formatted key structures. These are keys, not accounts. Even if an account associated with a key is inactive, deleted, or non-existent, the key itself can still be added to the FEKL.
```

Comment:
Is `key_verification_mode` meant to change this behavior?

Reply:
No, `key_verification_mode` only applies to keys on the topic that otherwise would have to sign the message, e.g. the `fee_schedule_key`.

Reply:
It would be useful to add an explanation of key_verification_mode to the hip text. It is present in the protobuf but not explained anywhere else.

Reply:
@kimbor is it ok for you if we postpone this note to a new PR, and we merge this in the meanwhile?
@netopyr can you take care of this additional note?

Reply:
Done